com.unity.netcode.gameobjects/Runtime/Core/NetworkManager.cs

```diff
@@ -485,6 +586,399 @@ private void OnValidate()
         }
 #endif
 
+        private bool ResolveNetworkPrefabs()
+        {
```

Comment:
The vast majority of this code is moved code, not code I wrote... I'm always hesitant to try to squeeze these kinds of refactors of other code into unrelated changes just because I moved it from one place to another. This all used to be part of Initialize and had to be delayed until assets are loaded.

Reply:
My initial reaction was similar to Matt's until I realized the majority of that came from the Initialize method.  What I finally "convinced myself" that this is "ok" for now was that there would be no real functional point in trying to break it up into several functions inside of NetworkManager since this is all NetworkPrefab initialization related code that really should live somewhere outside of NetworkManager but still invoked by NetworkManager.
Now that it is all in its own function it will be easier to create a small task after this PR to migrate the code into a "NetworkPrefab" relative location...perhaps even beak it up into smaller chunks during or after that PR. 
_(like 1-2 minor PRs)_